src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -888,11 +896,9 @@ public override void Accept(OperationVisitor visitor)
 
     internal partial class BoundBaseReference : IInstanceReferenceExpression
     {
-        bool IInstanceReferenceExpression.IsExplicit => this.Syntax.Kind() == SyntaxKind.BaseExpression;
-
-        IParameterSymbol IParameterReferenceExpression.Parameter => (IParameterSymbol)this.ExpressionSymbol;
+        InstanceReferenceKind IInstanceReferenceExpression.InstanceReferenceKind => InstanceReferenceKind.BaseClass;
```

Comment:
I ask because in IL there are more bindings possible than this/base/MyClass can imply and I'm not sure what to answer for such cases.